tck.sh

```diff
@@ -34,49 +34,53 @@ docker build -t $PYTHON_TCK_NAME ./
 docker network create $NETWORK_NAME
 
 #  primary tck tests for shopping cart
+docker run -d --network $NETWORK_NAME --name $USER_FUNCTION_NAME -p 8080:8080 $PYTHON_TCK_NAME server shoppingcart
 docker run -d --network $NETWORK_NAME --name $PROXY_NAME -p 9000:9000 \
     -e USER_FUNCTION_HOST=$TCK_NAME \
     -e USER_FUNCTION_PORT=8090 \
-    $PROXY_IMAGE
-sleep 10
-docker run -d --network $NETWORK_NAME --name $USER_FUNCTION_NAME -p 8080:8080 $PYTHON_TCK_NAME \
-    server \
-    shoppingcart
-sleep 10
+    $PROXY_IMAGE \
+    -e HTTP_PORT=9000
 docker run --rm --network $NETWORK_NAME --name $TCK_NAME -p 8090:8090 \
     -e TCK_HOST=0.0.0.0 \
+    -e TCK_PORT=8090 \
     -e TCK_PROXY_HOST=$PROXY_NAME \
-    -e TCK_FRONTEND_HOST=$USER_FUNCTION_NAME \
+    -e TCK_PROXY_PORT=9000 \
+    -e TCK_SERVICE_HOST=$USER_FUNCTION_NAME \
+    -e TCK_SERVICE_PORT=8080 \
     $TCK_IMAGE
 
-
 status=$?
+docker container logs $PROXY_NAME
+docker container logs $USER_FUNCTION_NAME
+
 echo "Removing docker containers"
 docker rm -f $PROXY_NAME
 docker rm -f $USER_FUNCTION_NAME
 
 #  secondary integration tests for stateless function:
 docker run -d --network $NETWORK_NAME --name $USER_FUNCTION_NAME -p 8080:8080 $PYTHON_TCK_NAME \
     server \
-    functiondemo \
+    ActionDemo \
     shoppingcart
-sleep 10
+sleep 5
```

Comment:
this races (alas did also before); but, can we `curl` that somehow perhaps?

Reply:
Use docker compose with dependency and probes. I think we could solve that problem with that approach. But we would start to depend on the docker-compose which I don't see as a problem because it is part of the Docker ecosystem. We don't need to do this in this PR but we must create an Issue to track this.

Reply:
issue created